### ISSUE TITLE
organization vdc are created only with SYSTEM org admin 

### ISSUE THREAD

**Issue by dev-dom** [vmware/ansible-module-vcloud-director#47]
Organization vdc are created only with SYSTEM org admin.
Looks like there is no way to use any other user.
The module should allow user to specify the organization admin not only the SYSTEM admin to create an organization vdc: this is critical to be implemented.
Thanks,
dom

**Comment by mukultaneja** [comment]
the module has not tied up with SYSTEM org or administrator user. You can update user/org in the relevant task playbook and module will use those details.

**Comment by dev-dom** [comment]
Thanks for your answer: however if I log directly to the org it won't be able to access to the pvdc details.
In fact I get this error:

`File "/usr/local/lib/python3.6/dist-packages/pyvcloud/vcd/org.py", line 1462, in create_org_vdc
    pvdc = system.get_provider_vdc(provider_vdc_name)
  File "/usr/local/lib/python3.6/dist-packages/pyvcloud/vcd/system.py", line 300, in get_provider_vdc
    'access to resource is forbidden' % name)
`

**Comment by dev-dom** [comment]
The module calls org.py and system.py that require SYSTEM as org value when logging in.
Anything different from SYSTEM will translate into error

**Comment by dev-dom** [comment]
Hello, I figure out where the bug is:

From https://github.com/vmware/ansible-module-vcloud-director/blob/master/modules/vcd_org_vdc.py

go to line 313->317

```
class Vdc(VcdAnsibleModule):
    def __init__(self, **kwargs):
        super(Vdc, self).__init__(**kwargs)
        logged_in_org = self.client.get_org()
        self.org = Org(self.client, resource=logged_in_org)
```

resource=logged_in_org returns an object related to client: client object authenticate as SYSTEM admin.

This causes the ovdc to be owned by SYSTEM (bug)

If instead of resource=logged_in_org you pass the href or the resource object of the org you want to create an ovdc, the module will correctly create the OVDC in the specified ORG.

Best Regards,
dom

**Comment by dev-dom** [comment]
I fixed the bug: will pull my code for review

**Comment by mgaruccio** [comment]
I encountered this problem as well, didn't see a PR from @dev-dom  so I submitted my fix in #53.

**Comment by mukultaneja** [comment]
Hi @mgaruccio @dev-dom,

Thanks for raising an issue and proposing the solution as well. I really apologize for delays in the implementation. I have merged your code with few other changes in the module. As I have added another operation and made few changes in the module so please allows me to close your merge request along with this issue.

Please validate the changes and feel free to reopen the issue. thanks!
